src/main/jhipster/src/main/webapp/app/germplasm-manager/germplasm-search.component.ts

```diff
@@ -807,11 +807,6 @@ export class GermplasmSearchComponent implements OnInit {
             return;
         }
 
-        if (this.size(this.selectedItems) > 500) {
-            this.alertService.error('germplasm-delete.too-many-selected-germplasm');
```

Comment:
Remove germplasm-delete.too-many-selected-germplasm entry in germplasm-manager.json